packages/mobile/src/features/videos/components/FiltersScreen/ShortcutToFilters.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { FontWeight } from '../../../../features/common/styles/constans/FontWeight';
+import { FontFamily } from '../../../../features/common/styles/constans/FontFamily';
+import { FontSize } from '../../../../features/common/styles/constans/FontSize';
+import { Color } from '../../../../features/common/styles/constans/Color';
+import BlackFilterButtonInShorcut from './BlackFilterButtonInShorcut';
+import { NavigationName } from '../../../../config/routing/NavigationName';
+import XButton from './XButton';
+import {MyText} from '../../../common/styles/MyText';
+;
+
+const Wrapper = styled.ScrollView({
+  flexGrow: 1,
+  backgroundColor: Color.WHITE,
+  shadowColor: Color.LIGHTGREYX,
+    shadowOffset: {
+        width: 1,
+        height: 1,
+    },
+    shadowOpacity: "1",
+    shadowRadius: 2,
+    elevation: "5",
+    flex: 1,
+    flexDirection: 'column',
+});
+
+const TitleEyeAndXWrapper = styled.View`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-top: 12px;
```

Comment:
ja używasz marginesu z każdej strony to ustawiałbym jako margin: cos cos cos cos

Reply:
racja, zmienione:     margin: 16px 12px 16px 8px;